source/atlas-cli-create-cluster-from-config-file.txt

```diff
@@ -0,0 +1,225 @@
+                           "diskIOPS": 3000,
+                           "ebsVolumeType": "STANDARD",
+                           "instanceSize": "M10",
+                           "nodeCount": 0
+                         },
+                         "electableSpecs": {
+                           "diskIOPS": 3000,
+                           "ebsVolumeType": "STANDARD",
+                           "instanceSize": "M10",
+                           "nodeCount": 3
+                         },
+                         "readOnlySpecs": {
+                           "diskIOPS": 3000,
+                           "ebsVolumeType": "STANDARD",
+                           "instanceSize": "M10",
+                           "nodeCount": 0
+                         },
+                         "autoScaling": {
+                           "diskGB": {
+                             "enabled": true
+                           },
+                           "compute": {
+                             "enabled": true,
+                             "scaleDownEnabled": true,
+                             "minInstanceSize": "M10",
+                             "maxInstanceSize": "M40"
+                           }
+                         },
+                         "priority": 7,
+                         "providerName": "AWS",
+                         "regionName": "US_EAST_1"
+                       }
+                     ]
+                   }
+                 ],
+                 "createDate": "2023-04-19T19:15:29Z",
+                 "rootCertType": "ISRGROOTX1",
+                 "versionReleaseSystem": "LTS",
+                 "terminationProtectionEnabled": false
+               }
+
+   .. step:: Remove the following settings in the configuration file.
+
+      - ``connectionStrings``
+      - ``id``
+      - ``replicationSpecs.id``
+      - ``createDate``
+
+   .. step:: Modify the cluster name (``name``) setting in the configuration file.
+
+      To learn more about the optional and required settings, see
+      :ref:`atlas-cli-cluster-config-file`. 
+
+   .. step:: Save and close the configuration file.
+
+Create a New Cluster Using the Configuration File 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. procedure::
+   :style: normal
+
+   .. step:: Connect to your |service| account for programmatic access if you aren't already connected to your |service| account. 
+
+      To learn more, see :ref:`connect-atlas-cli`.
+
+   .. step:: Run the following command to create an |service| cluster using the configuration file. 
+
+      .. code-block::
+         :copyable: true
+
+         atlas clusters create <new-cluster-name> -f myCluster.json
+
+      Replace ``<new-cluster-name>`` in the preceding command with the
+      name of the new {+cluster+} you wish to create. Note that the 
+      {+cluster+} name that you provide with the command must match the
+      value for the ``name`` setting in the configuration file. 
```

Comment:
FYI, we don't validate this, the name you provide as a command option will override the one in the file, you can use one or the other